Pet-Clinic-Frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react'
+import './Login.css'
+
+
+function LoginForm() {
+  const [username, setUsername] = useState("")
+  const [password, setPassword] = useState("")
+
+  return (
+    <>
+      <h1>Pet Clinic</h1>
+      <h2>Login</h2>
+     <div className = "formDiv">
+        <form className='form' onSubmit={function (event){
+          event.preventDefault()
+          console.log("Form submitted")
+          console.log('username:' , username)
+          console.log('password: ' , password)
+          window.location.reload()
```

Comment:
We probably we do not want to `console.log` and reloading things with `window.reload`.